hieradata/common.yaml

```diff
@@ -1311,6 +1311,8 @@ hosts::production::licensify::hosts:
     ip: '10.5.0.18'
 
 hosts::production::management::hosts:
+  db-admin-1:
```

Comment:
fair enough (we should probably look at rolling those tests out to cover stuff in `hieradata_aws` too but that's for another PR...)